ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrades.robot

```diff
@@ -42,21 +42,21 @@ ${DW_PROJECT_CREATED}=    False
 *** Test Cases ***
 Set PVC Size Via UI
     [Documentation]    Sets a Pod toleration via the admin UI
-    [Tags]  Upgrade
+    [Tags]  Upgrade    PreUpgrade
```

Comment:
to optimize, you could move the Tags at the beginning of the suite

Reply:
Thanks!! Done!!